lib/beaker-rspec/helpers/serverspec.rb

```diff
@@ -134,7 +134,7 @@ def self.method_missing(meth, *args)
     end
 
     def self.run(meth, *args)
-      backend = Specinfra.backend
+      Specinfra.backend
```

Comment:
This also just looks redundant since 2 lines below it again determines the backend
```suggestion
```